index.html

```diff
@@ -82,127 +91,175 @@ <h3>I’am a Software Developer</h3>
           </ul>
         </div>
       </div>
+      <div class="image-desktop">
+        <img src="imgs/Illustration-header.png" alt="background" />
+      </div>
       <div class="image">
         <img src="imgs/Illustration.png" alt="background" />
       </div>
     </section>
     <!--  headline section end -->
+
     <!--  work section start -->
-    <section id="recent-work">
+    <section class="recent-work">
       <div class="container">
         <div class="section-header">
           <h1>My Recent Work</h1>
```

Comment:
@MoRaad97 Kindly modify the `My recent work` section according to the content of the Figma template. Four cards do not resemble the content of the Figma template which I have circled in the below screenshot. 

![moharecentworkswrongedited](https://user-images.githubusercontent.com/87413774/165866605-bab34faa-a957-4df4-992f-b58c9a8033d2.png)

Below is the correct screenshot showing the correct content of the `My recent works` section

![moharecentworkscorrect](https://user-images.githubusercontent.com/87413774/165866704-4b5c382e-44cf-46f5-aff7-b88551151b47.png)